src/types/Plea.ts

```diff
@@ -0,0 +1,8 @@
+export enum Plea {
+  Guilty = 1,
+  NotGuilty = 2,
+  Consents = 4,
+  Guilty6 = 6,
```

Comment:
is guilty6 used anywhere?

Reply:
Good spot - this came from the data in Bichard and I added it for completeness. Basically, Guilty can be either `1` or `6`